modules/microshift-install-rhel-image-mode-conc.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-rhel-image-mode.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-rhel-image-mode-conc_{context}"]
+= Image mode for {op-system-base-full}
+
+Image mode for {op-system-base-full} is a technology preview deployment method that uses a container-native approach to build, deploy and manage the operating system as a bootc image. By using bootc, you can build, deploy, and manage the operating system as if it is any other container.
```

Comment:
```suggestion
Image mode for {op-system-base-full} is a Technology Preview deployment method that uses a container-native approach to build, deploy, and manage the operating system as a bootc image. By using bootc, you can build, deploy, and manage the operating system as if it is any other container.
```
You used the oxford comma in the last sentence, so I would use it on that second sentence too.